src/interface/TWUTXOPlanner.cpp

```diff
@@ -6,50 +6,13 @@
 
 #include <TrustWalletCore/TWUTXOPlanner.h>
```

Comment:
Let's merge TWUTXOPlanner.h/cpp into AnySigner to reduce another C interface (don't forget to remove Java/Swift related code)